modules/abstract-utxo/src/transaction/fixedScript/parseTransaction.ts

```diff
@@ -74,6 +74,16 @@ export async function parseTransaction<TNumber extends bigint | number>(
 
   const allOutputs = [...explanation.outputs, ...explanation.changeOutputs];
 
+  /** Get the amount on the transaction. This amount is being derived explicitly from the txHex
+   * and not by any API metadata, so this is safe to do */
+  function getExplanationAmountForAddress(address: string): number | string {
+    const recipient = explanation.outputs.find((output) => output.address === address);
```

Comment:
there can be more than one output with an address - this only returns the first one